packages/pds/src/api/com/atproto/sync/subscribeRepos.ts

```diff
@@ -27,27 +27,27 @@ export default function (server: Server, ctx: AppContext) {
           message: 'Requested cursor exceeded limit. Possibly missing events',
         }
       }
-      if (curr && cursor > curr.seq) {
+      if (curr && cursor > (curr.seq ?? 0)) {
```

Comment:
Just double-checking— cursors are always greater than `0`, sound right?

Reply:
changed this around a bit to avoid possible footguns

`if (curr?.seq && cursor > curr.seq) {`